doc/source/release/1.13.0-notes.rst

```diff
@@ -140,13 +140,38 @@ Deprecated features
 - Complex dtypes in ``PchipInterpolator`` and ``Akima1DInterpolator`` have
   been deprecated and will raise an error in SciPy 1.15.0. If you are trying
   to use the real components of the passed array, use ``np.real`` on ``y``.
-
+- Non-integer values of ``n`` together with ```exact=True`` are deprecated for
```

Comment:
I think there's a small blemish here too, the extra preceding backtick shows up in the rendered version of the relnotes:

https://scipy.github.io/devdocs/release/1.13.0-notes.html#deprecated-features

Pretty minor though